lte/gateway/python/integ_tests/s1aptests/test_ipv4v6_non_nat_ul_tcp.py

```diff
@@ -0,0 +1,133 @@
+"""
+Copyright 2020 The Magma Authors.
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import ipaddress
+import time
+import unittest
+
+import s1ap_types
+from integ_tests.s1aptests import s1ap_wrapper
+from s1ap_utils import MagmadUtil
+
+
+class TestIpv4v6NonNatUlTcp(unittest.TestCase):
+    """Integration Test: TestAttachDetachNonNatDpUlTcp"""
+
+    def __init__(self, method_name: str = ...) -> None:
+        """Initialize unittest class"""
+        super().__init__(methodName=method_name)
+        self.magma_utils = MagmadUtil(None)
+
+    def setUp(self):
+        """Initialize before test case execution"""
+        self.magma_utils.disable_nat(ip_version=6)
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper(ip_version=6)
+
+    def tearDown(self):
+        """Cleanup after test case execution"""
+        self._s1ap_wrapper.cleanup()
+        self.magma_utils.enable_nat(ip_version=6)
+
+    def test_ipv4v6_non_nat_ul_tcp(self):
+        """Basic ipv4v6 attach/detach and UL TCP data test with a single UE"""
```

Comment:
Any plans to add IPv6 data test cases for multi UE case

Reply:
Yes in future.